packages/libs/eda/src/lib/map/analysis/MarkerConfiguration/PieMarkerConfigurationMenu.tsx

```diff
@@ -68,6 +149,71 @@ export function PieMarkerConfigurationMenu({
         toggleStarredVariable={toggleStarredVariable}
         constraints={constraints}
       />
+      <div style={{ margin: '5px 0 0 0' }}>
+        <span style={{ fontWeight: 'bold' }}>
+          Summary marker (all filtered data)
+        </span>
+        {overlayConfiguration?.overlayType === 'categorical' ? (
+          <MarkerPreview data={overlayConfiguration} mapType="pie" />
+        ) : (
+          continuousMarkerPreview
+        )}
+      </div>
+      <LabelledGroup label="Donut marker controls">
```

Comment:
Need to make this render only for continuous vars?

Reply:
Maybe? Unclear it we'd rather disable them or remove them. Removing them has the benefit of uncluttering the UI but conversely results in page jumps and less consistency. Currently, I'm disabling binning controls in both marker config menus for categorical vars.